src/wp-content/themes/twentytwenty/functions.php

```diff
@@ -220,11 +220,9 @@ function twentytwenty_register_scripts() {
 		get_template_directory_uri() . '/assets/js/index.js',
 		array(),
 		$theme_version,
-		array(
-			'in_footer' => false, // Because involves header.
-			'strategy'  => 'defer',
-		)
+		false // Because involves header. The $args array is not used here to avoid unintentional footer placement in WP<6.3.
```

Comment:
I would prefer to omit the `false` here for consistency sake since it's the old signature, and just keep the comment instead - or pass an empty array?

Alternatively, we can be more verbose and do an `if else` for the entire function call, based on the WP version used.

Reply:
Done in 314e62faf3